src/server/page-renderer.js

```diff
@@ -91,10 +62,7 @@ class PageRenderer extends PageRendererBase {
 
     const styles = [{ href: '/main.css' }];
 
-    const scripts = [
-      { content: inlineScript },
-      { src: `/${bundleName}.js` }
-    ];
+    const scripts = [{ content: inlineScript }, { src: '/main.js' }];
```

Comment:
I won't pretend I'm completely clear on the bundle unification 😅 
For one I presume it was some overhead to keep them with the PageTemplate setup, but I guess it also makes sense since we're going in the direction of inline editing?
Or was there yet another thought behind this?

Reply:
Or actually it has to do with the `page-resolver.js`? 🤔 I will study it for some clarity here.

Reply:
We have now automatic code splitting, so it's not necessary anymore to manually create separate bundles, also it's getting harder and harder with each new page. It's still the case that each page uses different chunks, they just start all with `main.js` now.